src/locales/en/common.json

```diff
@@ -205,6 +205,9 @@
       "title": "Oops, insufficient balance",
       "description": "Make sure the account to debit has sufficient balance"
     },
+    "NotEnoughGas": {
+      "title": "Oops, insufficient gas"
```

Comment:
>_ETH balance in parent account insufficient for network fees_

According to https://github.com/LedgerHQ/ledger-live-desktop/blob/63b38a673c9d3d50b7997014557f37888b49dc7f/static/i18n/es/app.json#L1056

Reply:
🤷‍♂

Reply:
Yeah agree we need same wording. If we can change the wording, please also update the screenshot before we confirm with the team this is where we need the fix to be.